tests/foreman/api/test_subnet.py

```diff
@@ -497,6 +496,34 @@ def test_positive_list_parameters(self):
         :expectedresults: The satellite should display all the subnet
             parameters
         """
+        parameter = [{'name': gen_string('alpha'),
+                      'value': gen_string('alpha')}]
+        org = entities.Organization().create()
+        loc = entities.Location(organization=[org]).create()
+        org_subnet = entities.Subnet(
+            location=[loc],
+            organization=[org],
+            ipam=u'DHCP',
+            vlanid=gen_string('numeric', 3),
+            subnet_parameters_attributes=parameter,
+        ).create()
+        self.assertEqual(org_subnet.subnet_parameters_attributes[0]['name'],
+                         parameter[0]['name'])
+        self.assertEqual(org_subnet.subnet_parameters_attributes[0]['value'],
+                         parameter[0]['value'])
+        sub_param = entities.Parameter(
+            name=gen_string('alpha'),
+            subnet=org_subnet.id,
+            value=gen_string('alpha')
+        ).create()
+        sub = entities.Subnet(id=org_subnet.id).read()
+        params_list = [(param['name'], param['value'])
+                       for param in sub.subnet_parameters_attributes]
+        self.assertEqual(len(params_list), 2)
+        for tuple_ in params_list:
```

Comment:
We need better variable name than `tuple_`.
I understand that `params_list` contains list of tuples, but here we should refer to what it actually is from business perspective, not what data structure happens to hold it at the time.
(And BTW, since this is name and value pair, perhaps it should be a dict, not list of tuples?)

Reply:
Done. TY